backend/spotless/views/mobileviews/cleaningviews.py

```diff
@@ -16,8 +16,11 @@ class CleaningScheduleViewSet(viewsets.ModelViewSet):
     permission_classes = [IsAuthenticated]
     http_method_names = ['get', 'patch']
 
-    def get_queryset(self):
-        return super().get_queryset().filter(cleaner=self.request.user)
+    def list(self, request, *args, **kwargs):
+        queryset = CleaningSchedule.objects.filter(cleaners=request.user)
+        
+        serializer = CleaningScheduleSerializer(queryset, many=True)
+        return Response(serializer.data)
```

Comment:
is there a reason u changed this

Reply:
yea the original one sent a query set back but idk how convert from query set to array so I just changed the backend to send a list instead